test/functional/normalization/message_format_test.go

```diff
@@ -127,95 +127,4 @@ var _ = Describe("[Functional][LogForwarding][Normalization] tests for message f
 		Entry("should recognize an CRITICAL message", "critical", "CRITICAL:  Unable to connect to server"),
 		Entry("should recognize an DEBUG message", "debug", "level=debug found the light"),
 	)
-	It("should parse application log format correctly", func() {
-
-		// Log message data
-		message := "Functional test message"
-		timestamp := "2020-11-04T18:13:59.061892+00:00"
-		nanoTime, _ := time.Parse(time.RFC3339Nano, timestamp)
-
-		// Template expected as output Log
-		var outputLogTemplate = functional.NewApplicationLogTemplate()
-		outputLogTemplate.Timestamp = nanoTime
-		outputLogTemplate.Message = fmt.Sprintf("regex:^%s.*$", message)
-		outputLogTemplate.Level = "*"
-
-		// Write log line as input to fluentd
-		applicationLogLine := functional.NewCRIOLogMessage(timestamp, message, false)
-		Expect(framework.WriteMessagesToApplicationLog(applicationLogLine, 10)).To(BeNil())
-
-		// Read line from Log Forward output
-		raw, err := framework.ReadRawApplicationLogsFrom(logging.OutputTypeFluentdForward)
-		Expect(err).To(BeNil(), "Expected no errors reading the logs")
-
-		// Parse log line
-		var logs []types.ApplicationLog
-		err = types.StrictlyParseLogs(utils.ToJsonLogs(raw), &logs)
-		Expect(err).To(BeNil(), "Expected no errors parsing the logs")
-
-		// Compare to expected template
-		outputTestLog := logs[0]
-		Expect(outputTestLog).To(FitLogFormatTemplate(outputLogTemplate))
-	})
-
-	It("should parse application log format correctly if log message contains 'stdout','stderr' (Bug 1889595)", func() {
-
-		// Log message data
-		message := "Functional test stdout stderr message "
-		timestamp := "2020-11-04T18:13:59.061892+00:00"
-		nanoTime, _ := time.Parse(time.RFC3339Nano, timestamp)
-
-		// Template expected as output Log
-		var outputLogTemplate = functional.NewApplicationLogTemplate()
-		outputLogTemplate.Timestamp = nanoTime
-		outputLogTemplate.Message = fmt.Sprintf("regex:^%s.*$", message)
-		outputLogTemplate.Level = "*"
-
-		// Write log line as input to fluentd
-		applicationLogLine := fmt.Sprintf("%s stdout F %s $n", timestamp, message)
-		Expect(framework.WriteMessagesToApplicationLog(applicationLogLine, 1)).To(BeNil())
-		// Read line from Log Forward output
-		raw, err := framework.ReadRawApplicationLogsFrom(logging.OutputTypeFluentdForward)
-		Expect(err).To(BeNil(), "Expected no errors reading the logs")
-		// Parse log line
-		var logs []types.ApplicationLog
-		err = types.StrictlyParseLogs(utils.ToJsonLogs(raw), &logs)
-		Expect(err).To(BeNil(), "Expected no errors parsing the logs")
-		// Compare to expected template
-		outputTestLog := logs[0]
-		Expect(outputTestLog).To(FitLogFormatTemplate(outputLogTemplate))
-	})
```

Comment:
why are these tests deleted?

Reply:
reverted changes to these specific tests